Makefile

```diff
@@ -191,9 +191,16 @@ image-build: ## Build the EPP image using Docker Buildx.
 		$(LOAD) \
 		$(IMAGE_BUILD_EXTRA_OPTS) ./
 
+.PHONY: multi-platform-init
+multi-platform-init: ## Set up Docker Buildx for multi-platform builds.
+	@docker buildx inspect multiplatform >/dev/null 2>&1 || docker buildx create --name multiplatform --use
+	@docker run --privileged --rm tonistiigi/binfmt --install all
+	@docker buildx inspect --bootstrap
+
 .PHONY: image-push
 image-push: PUSH=--push ## Build the EPP image and push it to $IMAGE_REPO.
-image-push: image-build
+image-push: PLATFORMS=linux/amd64,linux/arm64
+image-push: multi-platform-init image-build
```

Comment:
I'd like to support both dev (single-arch) and release (multi-arch) workflows. [Here](https://gist.github.com/danehans/3b5622983ad0756bfe824a899a816822) is an untested example of what I was thinking to support multi-arch and support the different dev/release flows. @kfswain @ahg-g this does require updating the CI job, e.g. `MULTI=true`, to build multi-arch images.